packages/engine/src/scene/systems/SceneObjectSystem.tsx

```diff
@@ -159,7 +156,6 @@ export function setupObject(obj: Object3D, entity: Entity, forceBasicMaterials =
 }
 
 const groupQuery = defineQuery([GroupComponent])
-const updatableQuery = defineQuery([UpdatableComponent, CallbackComponent])
```

Comment:
Did you provide an alternative for this? We are currently using this in https://github.com/theinfinitereality/irpro-advanced-materials/ for animating materials for example the WaterStylizeMaterial and so this branch is breaking that functionality.

Reply:
You can just define a system. Pretty easy to fix.

Reply:
I believe the use case is that each individual material has it's own callback function. @dinomut1 first used useExecute I believe but was told not to democratically. But if it's gonna be removed it would be good to fix the dependency in the update PR for advanced materials. But personally I'd just revert the removal as it's a useful paradigm for now.

Reply:
we can just query for a component and call whatever updates we want. thats the point of an ECS. it's tech debt and not a pattern we should keep

Reply:
Not in scope to remove updateable component as part of a PR to migrate from cjs to esm. We do not have an agreed upon replacement for advanced materials, which are not components and thus cannot be queried.

Reply:
I don't believe this change is related to this ticket and is being done in an incorrect way. Historically we try to ensure a PR that is merged in won't break deployments. In its current state any deployment with our core packages installed would break. The best answer is to revert this change and to record a ticket for this work to be done seperatly when we have more bandwidth. Then we can remove this tech debt and refactor its many dependencies at the same time.